script/appsignal

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd "$(dirname "$0")/.."
+
+if [[ $DYNO == $APPSIGNAL_DYNO_ENABLED ]]; then
+    bundle exec appsignal install
```

Comment:
I believe the require is all that is needed to actually load appsignal in production. the install script is a way of interactively generating the config file. as per https://github.com/appsignal/appsignal-ruby/blob/master/lib/appsignal/cli/install.rb#L91